aiida_pseudo/data/pseudo/pseudo.py

```diff
@@ -18,23 +20,26 @@ class PseudoPotentialData(plugins.DataFactory('singlefile')):
     _key_md5 = 'md5'
 
     @classmethod
-    def get_or_create(cls, stream: typing.BinaryIO, filename: str = None):
+    def get_or_create(cls, source: typing.Union[str, pathlib.Path, typing.BinaryIO], filename: str = None):
         """Get pseudopotenial data node from database with matching md5 checksum or create a new one if not existent.
 
-        :param stream: a filelike object with the binary content of the file.
+        :param source: the source pseudopotential content, either a binary stream, or a `str` or `Path` to the absolute
```

Comment:
```suggestion
        :param source: the source pseudopotential content, either a binary stream, or a ``str`` or ``Path`` to the absolute
```

Also: Does the path have to be absolute? I did some field testing and it seemed to work just fine for relative paths?

Reply:
No, it shouldn't. The `SinglefileData` base class actually imposes this requirement, but I didn't see a good reason for it and so I made it such that it supports relative paths as well. If it is incorrect, it will simply raise `FileNotFoundError`. I will update the docstrings.